sensor/common/scan/scan_test.go

```diff
@@ -362,6 +363,21 @@ func (suite *scanTestSuite) TestEnrichThrottle() {
 	suite.Require().ErrorIs(err, ErrEnrichNotStarted)
 }
 
+func (suite *scanTestSuite) TestCentralDelegatedScanThrottle() {
+	ls := LocalScan{
+		scannerClientSingleton: emptyScannerClientSingleton,
+		scanSemaphore:          semaphore.NewWeighted(5),
+		adHocScanSemaphore:     semaphore.NewWeighted(0),
+	}
```

Comment:
The semaphore #'s aren't being tested here since the context is by default canceled, like test above this one, lets set a `maxSemaphoreWaitTime`.

```suggestion
	ls := LocalScan{
		scannerClientSingleton: emptyScannerClientSingleton,
		scanSemaphore:          semaphore.NewWeighted(5),
		adHocScanSemaphore:     semaphore.NewWeighted(0),
		maxSemaphoreWaitTime: 1 * time.Millisecond,
	}
```